desktop/cmp/panel/impl/ResizeContainer.js

```diff
@@ -37,12 +38,15 @@ export const resizeContainer = hoistCmp.factory({
         const cmp = vertical ? vbox : hbox,
             maxDim = vertical ? 'maxHeight' : 'maxWidth';
 
-        return cmp({
-            ref,
-            className,
-            flex: 'none',
-            [maxDim]: '100%',
-            items
+        return refreshContextView({
```

Comment:
My only concern here is that this refreshContext only exists for a resizable/collapsible panel. Maybe that is what we want, but we got burned by overly branching the structure between the two types of panel before.....

Reply:
Good point - will move to the main panel component, would be less surprising